config/generator-flags.json

```diff
@@ -0,0 +1,23 @@
+{
+    "@asyncapi/html-template": {
+        "flag": " -o example"
+    },
+    "@asyncapi/markdown-template": {
+        "flag": " -o example"
+    },
+    "@asyncapi/nodejs-template": {
+        "flag": " -p server=production -o example"
+    },
+    "@asyncapi/nodejs-ws-template": {
+        "flag": " -p server=production -o example"
```

Comment:
this actually won't work as the https://bit.ly/asyncapi file is not for websocket 🤔 
might be that https://bit.ly/asyncapi needs to be configurable the same way flags are